README.md

```diff
@@ -1,64 +1,41 @@
-# 💩Shitty💩 Sync
+# Shitty Sync
 
-[![Docker Image CI](https://github.com/MaticBabnik/shitty-sync/actions/workflows/docker-image.yml/badge.svg)](https://github.com/MaticBabnik/shitty-sync/actions/workflows/docker-image.yml)
+[![Docker Image CI](https://github.com/MaticBabnik/shitty-sync/actions/workflows/docker-images.yml/badge.svg)](https://github.com/MaticBabnik/shitty-sync/actions/workflows/docker-images.yml)
 
 A (somewhat) shitty way to sync videos with your friends.
 
-Check out a demo [**sync.si instance**](https://sync.si/)!
+Live @ [**sync.si**](https://sync.si/)!
 
-## Getting started
+## Deploying using Docker Compose:
 
-1. Pull this repo
-2. Install the dependencies with `npm i` (both in the root folder & in the `frontend` folder)
-3. Build the frontend with `npm run build`
-4. Run the server with `npm run start`
+1. Copy the compose file
+2. `docker compose up -d`
 
-## 📈 Metrics
-
-The server enables access to performance metrics using Prometheus. They are exposed by default on the `/metrics`
-endpoint.
-
-## 🌳Environment variables
-
-var            | Description
--------------- | -----------
-`PORT`         | Webserver port, defaults to 8080
-`PORT_METRICS` | Prometheus `/metrics` port, defaults to 9090 and should not be the same as PORT
-`NO_METRICS`   | Disables metrics if set to a [truthy value](https://developer.mozilla.org/en-US/docs/Glossary/Truthy)
-
-## Deploying using 🐳 Docker:
+need I say more?
```

Comment:
Let's be more professional next time ok?

Reply:
SIMPLE!